### DISCUSSION TITLE
Save the value of a response

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5954] [Q&A] [answered]
Hello! 

I have 1 channel that sends authentication credentials, and on success the process will return a token, the question is: How can I take the value of the response token? I need to convert it into a GlobalMap.

![image](https://github.com/nextgenhealthcare/connect/assets/98954172/9b47fb90-aacb-4beb-821a-d3772964281d)

**Comment by jonbartels** [comment, marked as answer]
Edit the response transformer. You can parse the JSON response there and put the token into a map. https://docs.nextgen.com/bundle/Mirth_User_Guide_41/page/connect/connect/topics/c_Response_TransformersResponse_Transformers_connect_ug.html

**Reply by 5011-5011** [reply]
Thank you very much!
